packages/aws-cdk/test/version.test.ts

```diff
@@ -77,9 +78,49 @@ test('No Version specified for storage in the TTL file', async () => {
 });
 
 test('Skip version check if environment variable is set', async () => {
-  process.stdout.isTTY = true;
-  process.env.CDK_DISABLE_VERSION_CHECK = '1';
+  sinon.stub(process, 'stdout').value({ ...process.stdout, isTTY: true });
+  sinon.stub(process, 'env').value({ ...process.env, CDK_DISABLE_VERSION_CHECK: '1' });
```

Comment:
I changed this test from directly setting these global "process" variables to stubbing them with sinon. This ensures that the values get set back to their original values after the tests run. Before this change, the `CDK_DISABLE_VERSION_CHECK` environment variable was still set in the environment, causing my new tests below to fail.